### PR TITLE
feature(plugins): improved plugin listing

### LINKED ISSUE #4158: turn plugin screenshots into gallery (Trac #4158)
_Original ticket http://trac.elgg.org/ticket/4158 on 41908480-12-06 by cash, assigned to unknown._

Elgg version: 1.8

I'd like to have next/previous buttons rather than opening/closing the lightbox for every screenshot

### LINKED ISSUE #8412: Pull ColdTrick/plugin_manager improvements into core
Nice work. https://github.com/ColdTrick/plugin_manager

@jdalsem @jeabakker

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #8412

Also fixes #4158, #4301, #6778 
- [ ] remove sorting actions
- [ ] remove crazy (de)activate all forms
- [ ] correct styling of filter buttons

### CODE REVIEW

**Comment by jdalsem** [comment]
Looking for feedback

**Comment by jdalsem** [comment]
naamloos

sample screenshot

File: views/default/object/plugin/details.php [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+	$tabs[] = array(
+		'text' => elgg_echo("admin:plugins:label:screenshots"),
+		'rel' => 'elgg-plugin-details-screenshots'
+	);
+}
+
+$tabs[] = array(
+	'text' => elgg_echo("admin:plugins:label:dependencies"),
+	'rel' => 'elgg-plugin-details-dependencies',
+	'selected' => (!$can_activate) ? true : false
```
Comment (ewinslow) [inline views/default/object/plugin/details.php]: This ternary makes me giggle :)

**Comment by ewinslow** [comment]
This UX is amazingly better and the code quality hasn't dropped dramatically or anything so far as I can tell.

LGTM.

Any BC implications here?

**Comment by jdalsem** [comment]
There is still some code to be cleaned up, as it is mainly an adapted version of the plugin_manager plugin. BC issue are there as there are changes in css and views, so that is why i did it on master.

Also on a side note... i dropped the sorting feature (by date/newest, active/inactive). Active/inactive are now filters and the other sorting option is hardly used.

**Comment by mrclay** [comment]
LGTM. Would be nice to tell when a plugin was added and activated, and by whom, but that sounds like another feature.

**Comment by ewinslow** [comment]
I'd say add those last 3 "fixes" bugs to the commit message and then pull this sucker in. We can tweak and clean up later as needed.

**Comment by ewinslow** [comment]
BTW each of those probably needs to go on its own line like:

```
Fixes #123
Fixes #456
Fixes #789
```

Whenever we've done `Fixes #476, #127, and #683` in the past, the 127 and 683 ones don't get closed correctly.

**Comment by mrclay** [comment]
Trying to finish this up in #8528

**Pull request closed without merging**